source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Autodefine External Network
+category: feature
+authors: Ales Musil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+Engine automatically creates an external network based on physical VM network
+with physical connection to it. 
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Autodefine helps reducing administrative tasks needed with OVS switch type in 
+the cluster.
```

Comment:
[Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) is useful for users who would like to manage their network via and SDN (such as OVN or Neutron) yet have near-native performance. However, without auto-define feature, such admin would have to create a physical network (and attach it to hosts), create an external network, and remember to connect the two together. 
Autodefine helps reducing these administrative tasks. In an OvS cluster with an external provider attached, the external network is automatically created and connected to the physical one.

Reply:
Done